populate/src/main/java/bio/terra/pearl/populate/service/extract/SiteContentExtractor.java

```diff
@@ -32,8 +32,9 @@ public void writeSiteContents(Portal portal, ExtractPopulateContext context) {
         List<SiteContent> siteContents = context.isExtractActiveVersionsOnly()
                 ? siteContentService.findActiveContentByPortalId(portal.getId())
                 : siteContentService.findByPortalId(portal.getId());
+
         for (SiteContent siteContent : siteContents) {
-            siteContentService.attachChildContent(siteContent, "en");
+            siteContentService.attachAllChildContent(siteContent);
```

Comment:
heh, I'd forgotten about that bit of hardcoding -- nice fix

Reply:
I did too - for 2 minutes, I was like, what the hell is happening??? where did all the content go???